lib/src/components/layout/accordion.dart

```diff
@@ -63,20 +203,35 @@ class _AccordionItemState extends State<AccordionItem>
   late AnimationController _controller;
   late CurvedAnimation _easeInAnimation;
 
+  bool _initialized = false;
+
   @override
-  void initState() {
-    super.initState();
-    // _expanded = widget.expanded;
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+
+    _AccordionState newAccordion = Data.of<_AccordionState>(context);
+    if (newAccordion != accordion) {
+      accordion?._expanded.removeListener(_onExpandedChanged);
+      newAccordion._expanded.addListener(_onExpandedChanged);
+      accordion = newAccordion;
+    }
+
+    if (_initialized) return;
+
+    _initialized = true;
+
+    final theme = ComponentTheme.maybeOf<AccordionItemTheme>(context);
```

Comment:
I don't think it should be block with initialized variable.
For example, the first time it gets mounted, it will initialize an animation controller with the duration from the theme or default value. But then when the theme changed, the animation controller would not update the duration.

Heres what I think you should do,
```suggestion
    final newTheme = ComponentTheme.maybeOf<AccordionItemTheme>(context);
    // declare a "theme" variable inside the component state
    if (newTheme != compTheme) {
      _controller?.dispose(); // dispose previous animation, would have to make the _controller variable a nullable instead of late
      compTheme = newTheme
      _controller = AnimationController(
        vsync: this,
        duration: theme?.duration ?? const Duration(milliseconds: 200),
```
_* also dispose and update the curved animation_

See line 212 - 216 for example.
Let me know if you need more info.